src/components/CountryData.js

```diff
@@ -0,0 +1,39 @@
+import { useSelector } from 'react-redux';
+import '../styles/details.css';
+import { NavLink } from 'react-router-dom';
+
+const CountryData = () => {
+  const store = useSelector((store) => store.homeReducer);
+  const { states, country } = store.details;
+  return (
+    <section>
+      <div className="board flex col">
+        {' '}
+        <NavLink to="/" className="homekey"><i className="fa-solid fa-house home" /></NavLink>
```

Comment:
- Good job including this button to route back to the homepage, but according to the project requirements, this button should be a `<` (back) button. Please fix this 🙏 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏